Sources/RunnerLib/HelpMessagePresenter.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  HelpMessagePresenter.swift
+//  RunnerLib
+//
+//  Created by Franco Meloni on 23/11/2018.
+//
+
+import Logger
+
+public final class HelpMessagePresenter {
+    public static func showHelpMessage(command: DangerCommand?, logger: Logger) {
```

Comment:
I passed the command here then we will be able to show the command specific help if any

Reply:
👍